projects/kit/components/rating/rating.template.html

```diff
@@ -0,0 +1,35 @@
+<label
+    #parent
+    class="t-rating"
+    [attr.aria-label]="value + ' Stars'"
```

Comment:
i18n

Reply:
I don't think we need it. Let's start with as little extra stuff as possible and we can always add it later if the need arises.